File: GT-web/src/app/services/conexion-bd.service.ts
import { HttpClient } from '@angular/common/http';
import { Injectable } from '@angular/core';
import { ConexionBD } from '../models/entities/ConexionBD';

@Injectable({
  providedIn: 'root'
})
export class ConexionBDService {
  private url: string;
  constructor(private http: HttpClient) {
    this.url = 'http://localhost:8080/gt-webcore/api/v1/bd/';
  }

  conexionBD(conexion: ConexionBD) {
    return this.http.post<any>(this.url + "conexion", conexion);
  }
}

File: GT-webcore/src/main/java/com/novatronic/pscabas/gt/webcore/services/ConexionBDService.java
package com.novatronic.pscabas.gt.webcore.services;

import java.sql.SQLException;

import com.novatronic.pscabas.gt.webcore.domains.entities.ConexionBD;

public interface ConexionBDService {

	public boolean conexionBD(ConexionBD conexion) throws SQLException;
}
